cypress/integration/home-page.spec.js

```diff
@@ -0,0 +1,240 @@
+
+        it('should navigate you to the events page when the view all events button is selected', () => {
+            cy.visit(constants.rootUrl);
+
+            cy
+                .wait(1000)
+                .get(constants.selectors.viewAllEventsButtons)
+                .first()
+                .click();
+
+            assertLocationChanged();
+        });
+    });
+
+    describe('From the Blog', () => {
+        before(() => {
+            cy.visit(constants.rootUrl);
+        });
+
+        it('should show four blog posts', () => {
+            cy
+                .get(constants.selectors.blogPost)
+                .its('length')
+                .should('eq', 4);
+        });
+
+        it('should navigate to a specific blog post', () => {
+            cy
+                .get(constants.selectors.blogPost)
+                .first()
+                .find('a')
+                .click();
+
+            assertLocationChanged();
+        });
+
+        it('should navigate a list of blog posts when view the blog is selecoted', () => {
+            cy.visit(constants.rootUrl);
+
+            cy.get(constants.selectors.viewTheBlogButton).click();
+
+            assertLocationChanged();
+        });
+    });
+
+    describe('Recommended Titles', () => {
+        let visibleTitles;
+        
+        const assertVisibleTitlesChanged = titleElms => {
+            Array.prototype.slice.call(titleElms).forEach(titleElm => {
+                const isMatch = visibleTitles.includes(titleElm.innerText);
+                expect(isMatch).to.be.equal(false);
+            });
+        };
+
+        const getActiveTab = () =>
+            cy
+                .get(constants.selectors.recommendedTitleContainer)
+                .find(constants.selectors.activeTab);
+
+        const getCarouselArrow = () =>
+            cy
+                .get(constants.selectors.bookCarousel)
+                .first()
+                .find(constants.selectors.recommendedTitlesArrows);
+
+        const getVisibleTitles = callback => {
+            cy
+                .get(constants.selectors.bookCarousel)
+                .first()
+                .find('p:visible')
+                .then(callback);
+        };
+
+        const setVisibleTitles = titleElms => {
+            Array.prototype.slice.call(titleElms).forEach(titleElm => {
+                visibleTitles.push(titleElm.innerText);
+            });
+        };
+
+        before(() => {
+            cy.visit(constants.rootUrl);
+        });
+
+        beforeEach(() => {
+            visibleTitles = [];
+        });
+
+        it('should initally have the fiction tab selected', () => {
+            getActiveTab().contains('Fiction');
+        });
+
+        it('should allow users to move through titles using the previous arrow in the control', () => {
+            getVisibleTitles(setVisibleTitles);
+
+            getCarouselArrow()
+                .first()
+                .click()
+                .wait(500);
```

Comment:
I am not sure about this but have you tried leaving off `.wait(500)`? The [docs](https://docs.cypress.io/guides/core-concepts/interacting-with-elements.html#Animations) say that cypress automatically waits for animations to finish.

Reply:
per our chat, going to leave this, because it's checking a different element.